src/deepsparse/engine.py

```diff
@@ -140,6 +140,11 @@ def _validate_scheduler(scheduler: Union[None, str, Scheduler]) -> Scheduler:
             "unsupported type for scheduler: {} ({})".format(scheduler, type(scheduler))
         )
 
+    if scheduler == Scheduler.elastic:
+        _LOGGER.warn(
```

Comment:
Could we use python's inbuilt `warnings` module instead of a LOGGER,  it is the recommended way after Python 3.6 and gives a nicer message, the code would look something like:
`import warnings`
`warnings.warn("Scheduler.elastic is an alias for Scheduler.multistream", DeprecationWarning, stacklevel=2)`


And the actual warning would look like:
```bash
/XXXXXX/projects/deepsparse/local-work.py:8: DeprecationWarning: Scheduler.elastic is an alias for Scheduler.multistream
```

Reply:
My bad, should be a `FutureWarning` instead

Reply:
good call, thank you